migrator/tests/actions/drafts/conftest.py

```diff
@@ -0,0 +1,1976 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Migrator tests configuration."""
+
+import pytest
+from invenio_rdm_migrator.load.postgresql.transactions.operations import OperationType
+
+
+@pytest.fixture()
```

Comment:
They need to be function or do a deep copy since the dict gets modified. Most of the transformations are done in place, maybe not in the root dict but in some inner dicts. I think this change would improve a bit test performance, but I dont think is worth the time it requires investing in _hunting_ down all the functions that do in-place modifications. 

Extra info: I think tests were slow in my laptop for some reason (docker, or similar). In GH actions they take 20s, and now in my laptop again. When I mentioned I could not test in GH actions due to missing dependencies releases. I think we can "shelve" the performance topic.